ko/lessons/advanced/otp-supervisors.md

```diff
@@ -6,15 +6,15 @@ order: 5
 lang: ko
 ---
 
-슈퍼바이저(Supervisor)는 다른 프로세스를 감시한다는, 단 하나의 목적에 특화된 프로세스입니다. 자식 프로세스가 실패하면 자동으로 재시작해주므로 장애에 대한 내성이 높은(Fault-tolerant) 애플리케이션을 만들 수 있게 해줍니다.
+슈퍼바이저(Supervisor)는 다른 프로세스의 감시라는 단 하나의 목적에 특화된 프로세스입니다. 자식 프로세스가 실패하면 자동으로 재시작해주는 것으로 장애에 대한 내성이 높은(Fault-tolerant) 애플리케이션을 만들 수 있게 해줍니다.
 
 {% include toc.html %}
 
 ## 설정
 
-슈퍼바이저의 마법은 `Supervisor.start_link` 함수의 내부에 있습니다. 슈퍼바이저 프로세스를 실행하면서, 슈퍼바이저가 자식을 관리하기 위해 사용할 전략을 정의할 수 있습니다.
+슈퍼바이저의 마법은 `Supervisor.start_link` 함수의 내부에 있습니다. 슈퍼바이저와 그자식 프로세스를 실행하면서, 슈퍼바이저가 자식 프로세스를 관리하기 위해 사용할 전략을 정의할 수 있습니다.
```

Comment:
@malkoG Your suggestion seems to be applied. Do you like these changes? If so, Can you make your review as **approved**? 😀